no-bid/no-bid-htb-system-tests.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+function getPartnerId() {
+    return 'NoBidHtb';
+}
+
+function getStatsId() {
+    return 'NOB';
+}
+
+function getCallbackType() {
+    return 'NONE';
+}
+
+function getArchitecture() {
+    return 'SRA';
+}
+
+function getConfig() {
+    return {
+        siteId: {
```

Comment:
The validator is expecting `siteId` to be a number. However, here we are passing in an object. Can you update the config to reflect the correct value ?

Reply:
I changed the getConfig() function to return siteId as a number instead of an object.

Reply:
@ix-certification Hello.
Thank you for looking into the root cause of the error. Can you please check the change we made yesterday?

-Regards

Reply:
Hello
No problem. The changes look good.